components/server/src/user/user-controller.ts

```diff
@@ -161,9 +161,10 @@ export class UserController {
                     });
                 });
 
-                // Redirect the user to create a new Organization
-                // We'll want to be more specific about the redirect based on the cell information in the future.
-                res.redirect("/orgs/new", 307);
+                // Redirect the admin-user to the Org Settings page.
+                // The dashboard is expected to render the Onboading flow instead of the regular view,
+                // but if the browser is reloaded after completion of the flow, it should be fine to see the settings.
+                res.redirect("/settings", 307);
```

Comment:
Afaik, this is not what the user sees after they complete the SSO setup. It's what the admin credentials redirect to but then it gets hijecked by the UI anyway to perform the setup.

Is this really the place to make this change? Shouldn't it be done client-side?